code/modules/surgery/tools.dm

```diff
@@ -105,7 +105,6 @@
 	hitsound = 'sound/weapons/circsawhit.ogg'
 	materials = list(/datum/material/iron=10000, /datum/material/glass=6000)
 	flags_1 = CONDUCT_1
-	force = 15
```

Comment:
For reference 15 damage is the damage of a welder when lit. Would consider lowing this one to toolbox damage at worst (12)

Reply:
Also isn't this going to be reducing it to 0 damage?

Reply:
(which will also mean no damage taken if you fail surgery)